cmd/crc/cmd/daemon.go

```diff
@@ -191,6 +204,45 @@ func run(configuration *types.Configuration) error {
 	}
 }
 
+// This API is only exposed in the virtual network (only the VM can reach this).
+// Any process inside the VM can reach it by connecting to gateway.crc.testing:80.
```

Comment:
Hmm, any process in the VM can fill the host root filesystem, and maybe slow down name resolution? (dunno if huge /etc/hosts has a noticeable impact on name resolution). I did not check the admin-helper code, but we can limit the number of aliases we add to a few hundreds, and also limit their individual size (which is done already I think), which would address this concern.

Reply:
https://github.com/code-ready/admin-helper/issues/14 added